src/main/java/online/book/store/dto/user/request/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package online.book.store.dto.user.request;
+
+import jakarta.validation.constraints.AssertTrue;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+import org.hibernate.validator.constraints.Length;
+
+@Data
+public class UserRegistrationRequestDto {
+    @NotBlank
+    @Email
+    private String email;
+    @NotBlank
+    @Length(min = 6)
+    private String password;
+    @NotNull
```

Comment:
```suggestion
    @NotBlank
```
Use `@NotBlank` for String. Also, check other fields.